test/cnxml/de/inline.js

```diff
@@ -12,7 +12,8 @@ export const input = cnxml`
 links to other elements (<link target-id="f1" />),
 elements in other documents (<link target-id="f1" document="d1" />),
 other <link document="d1">documents</link>,
-and <link url="https://example.test">external links</link>.</para>
+and <link url="https://example.test">external links</link>
+<footnote id="footnote-id">Footnote</footnote>.</para>
```

Comment:
If you put `<footnote>` before `<link>` you'll modify just one line, and keep this sentence grammatically correct.